### PR TITLE
Fix: Remove "Sign up with GitHub" button on mobile

### LINKED ISSUE #426: [Bug] Remove "Sign up with GitHub" button on mobile
**To Reproduce**
View app on mobile.

**Expected behavior**
No "Sign up with GitHub" button

**Actual behavior**
It's there.

**Notes**

### PR DESCRIPTION
**Pull request by Jadhielv**
## Description
Ability to hide "Sign up with GitHub" button on mobile. Created new conditional to include an `isMobile` check.

Closes #426

### Browser checklist

This PR has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Safari

### Testing checklist

- [ ] End-to-end tests have been created if necessary

### CODE REVIEW

**Comment by taniarascia** [comment]
Can you please refactor the logic here. It does not make sense to check for isMobile in one spot and !isMobile in another spot. This can be simplified. Thanks!

**Comment by Jadhielv** [comment]
Done @taniarascia

File: src/client/components/LandingPage.tsx [hunk 1]
```diff
@@ -32,23 +32,24 @@ export const LandingPage: React.FC = () => {
...
-                    and <b>not</b> persisted in any database or cloud.
-                  </p>
-                  <a className="button" href="/app">
-                    View Demo
-                  </a>
-                </div>
-              ) : (
-                [
-                  !isMobile ? (
+            ) : (
```
Comment (taniarascia) [inline src/client/components/LandingPage.tsx:37]:
```js
: (
  [
    isDemo ? (
```

Can be written as:

```js
: isDemo ?
```

Comment (Jadhielv) [reply src/client/components/LandingPage.tsx:49]: Done.

**Pull request merged**